Wyvarn/calamus
# Fetch Public profile of user by ID

## Description

This allows making a request to fetch a use by their ID

Fixes #14 

## How can we test this?

Run application with `docker-compose up` & `yarn start` or `npm run start`
Ensure you already have registered users

Make a request to get a user with their id as below

```bash
curl --request GET \
  --url http://localhost:3000/api/v1/profile/public/id/5e9c1b43b3df6667d15c5f25 \
  --header 'content-type: application/json'
```

Response should be:

```json
{
	"statusCode": "10000",
	"message": "success",
	"data": {
		"name": "<NAME>",
		"profilePicUrl": "<PIC>"
	}
}
```

## Checklist:
